.circleci/config.yml

```diff
@@ -36,10 +36,11 @@ jobs:
       - run: sudo apt-get update
       - run: sudo apt-get install tree
       - run: sudo apt-get install cf-cli
+      - run: cf install-plugin https://github.com/govau/autopilot/releases/download/0.0.5-venapp/autopilot-linux -f
       - run: cf version                # log what version we are running
       - run: cf login -a api.system.y.cld.gov.au -o $CF_ORG_STAGING -s $CF_SPACE_STAGING -u $CF_USER_STAGING -p $CF_PASSWORD_STAGING
       - run: tree -C                   # log all files before we push
-      - run: cf push -f manifest-staging.yml
+      - run: cf zero-downtime-push -f manifest-staging.yml
```

Comment:
This needs to specify an app name, e.g. `cf zero-downtime-push service-manual -f manifest-staging.yml`

Reply:
ah, ok.  i'd _assumed_ that listing the application in manifest-*.yml would work.  I'll update.  thanks.